runtime/common/src/migrations.rs

```diff
@@ -80,6 +83,46 @@ where
 	}
 }
 
+pub struct RemoveMinBondForOrbiterCollators<T>(pub PhantomData<T>);
+impl<T> Migration for RemoveMinBondForOrbiterCollators<T>
+where
+	T: pallet_moonbeam_orbiters::Config,
+	T: pallet_parachain_staking::Config,
+	T: frame_system::Config,
+{
+	fn friendly_name(&self) -> &str {
+		"MM_RemoveMinBondForOrbiterCollators"
+	}
+
+	fn migrate(&self, _available_weight: Weight) -> Weight {
+		let mut weight = Weight::zero();
+		CollatorsPool::<T>::iter_keys().for_each(|collator| {
+			log::info!("Setting the bond for collator {:?} to zero", collator);
+			weight += <pallet_parachain_staking::Pallet<T>>::set_candidate_bond_to_zero(&collator)
+				.expect("failed to set collator bond to 0")
+				.actual_weight
+				.expect("failed to get weight");
+		});
+		weight
+	}
+
+	#[cfg(feature = "try-runtime")]
+	fn pre_upgrade(&self) -> Result<Vec<u8>, sp_runtime::DispatchError> {
+		Ok(vec![])
+	}
+
+	#[cfg(feature = "try-runtime")]
+	fn post_upgrade(&self, _state: Vec<u8>) -> Result<(), sp_runtime::DispatchError> {
+		CollatorsPool::<T>::iter_keys().for_each(|collator| {
+			log::info!("Checking collator: {:?}", collator);
+			let state = <pallet_parachain_staking::Pallet<T>>::candidate_info(&collator)
+				.expect("collator should have candidate info");
+			assert!(state.bond.is_zero(), "collator bond should be zero");
```

Comment:
To be even more pedantic, I would check that there isn't a `COLLATOR_LOCK_ID` lock for the collator. In theory, this should never exist after this PR (and its migration), unless I'm misunderstanding.